src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/TriviaEngine/AbstractTriviaFormatter.cs

```diff
@@ -279,27 +289,41 @@ private LineColumn FormatTrivia<T>(Formatter<T> formatter, WhitespaceAppender<T>
             var implicitLineBreak = false;
 
             var list = new TriviaList(this.Token1.TrailingTrivia, this.Token2.LeadingTrivia);
+            var i = -1;
+            var previousLineColumn = new LineColumn();
             foreach (var trivia in list)
             {
+                i += 1;
                 if (trivia.RawKind == 0)
                 {
                     continue;
                 }
 
                 if (IsWhitespaceOrEndOfLine(trivia))
                 {
+
+                    existingWhitespaceDelta = existingWhitespaceDelta.With(
+                       GetLineColumnOfWhitespace(
+                           lineColumn,
+                           previousTrivia,
+                           previousWhitespaceTrivia,
+                           existingWhitespaceDelta,
+                           trivia));
+
                     if (IsEndOfLine(trivia))
                     {
                         implicitLineBreak = false;
+                        previousLineColumn = new LineColumn();
+                    }
+                    else
+                    {
+                        if (IsLineContinuation(previousTrivia)
+                           && (i + 1) < list.Count
+                           && IsComment(list[i + 1]))
```

Comment:
❔ What's the difference between `list[i + 1]` and `trivia`?

Reply:
Trivia is list(i) this is a look ahead. I need to look back, for underscore and forward for comment while list(i) is on WhitespaceOrEndOfLine to capture the current indent. Underscore whitespace comment needs to treated as a singe item for formatting and not a reason to shift line right.

Reply:
Does VB support `_' Comment` (i.e. no space between `_` and `'`)?

Reply:
No at least 1 space is required between _ and ', BUT there is an autocorrect if you leave it out and this code should still supports that the way it is written.